packages/env/README.md

```diff
@@ -183,6 +183,26 @@ Out of the box `wp-env` includes the [WordPress' PHPUnit test files](https://dev
 
 While we do provide a default `wp-tests-config.php` file within the environment, there may be cases where you want to use your own. WordPress provides a `WP_TESTS_CONFIG_FILE_PATH` constant that you can use to change the `wp-config.php` file used for testing. Set this to a desired path in your `bootstrap.php` file and the file you've chosen will be used instead of the one included in the environment.
 
+## Using `composer`, `phpunit`, and `wp-cli` tools.
+
+For ease of use, Composer, PHPUnit, and wp-cli are available for in the environment. To run these executables, use `wp-env run <env> <tool> <command>`. For example, `wp-env run cli composer install`, or `wp-env run tests-cli phpunit`. You can also access various shells like `wp-env run cli bash` or `wp-env run cli wp shell`.
```

Comment:
I worry that including `wp-cli` here makes it ambiguous. It feels like this is saying you should use `wp-env run cli wp-cli <command>`.

Reply:
That's a good point. I was trying to provide a search result if someone is looking to use `wp-cli`. And to actually use it, you have to do `run cli wp`... how do you think we should phrase it?

Reply:
Actually, I think this is fine. If you look at https://wp-cli.org, the official documentation has you using `wp` as the command. Anyone looking to use `wp-cli` will have that as the expectation I bet.